testing_data/model_config.json

```diff
@@ -1,57 +1,144 @@
 {
 	"command": "eval",
 	"gpu": 1,
-        "target_suffix": "_lesion-manual",
-        "roi_suffix": "_seg-manual",
-	"bids_path": "../duke/sct_testing/large/",
+	"log_directory": "testing_script",
+	"debugging": false,
+    "object_detection_params": {
+        "object_detection_path": null
+    },
+	 "loader_parameters": {
+		 "target_suffix": "_lesion-manual",
+		 "roi_suffix": "_seg-manual",
+		 "bids_path": "../duke/sct_testing/large/",
+		 "roi_params": {
+            "suffix": null,
+            "slice_filter_roi": null
+        },
+
+		"contrast_params": {
+			"random_seed": 1313,
+			"training_validation": [
```

Comment:
We could reduce this list one day :-D Given we only have one testing img.

Reply:
done in 8f35943